docs/modelagem/prototipo_alta.md

```diff
@@ -0,0 +1,63 @@
+# Protótipo de Alta fidelidade
+ 
+## 1.Introdução
+Segundo o artigo "Prototipagem de alta fidelidade: o que é, quando, por que e como usar?", escrito pelo instituto de educação Tera, um protótipo de alta fidelidade (às vezes chamado de high-fi ou hi-fi) é uma representação interativa do produto, baseada no computador ou em dispositivos móveis. Esse protótipo já apresenta maior semelhança com o design final em termos de detalhes e funcionalidade.
+ 
+O “high” em alta fidelidade refere-se ao nível de abrangência, que permite examinar questões de usabilidade em detalhes e tirar conclusões sobre o comportamento da pessoa usuária que testa o protótipo.
+ 
+Os protótipos de alta fidelidade abrangem não apenas a interface da pessoa usuária (UI) do produto em termos visuais e estéticos, mas também os aspectos da experiência do(a) usuário(a) (UX) em termos de interações, fluxo e comportamento.
+ 
+## 2.Metodologia
+Para desenvolver o protótipo de alta fidelidade, os integrantes do grupo Estevão Reis, Pedro Haick e Tomás Veloso, se reuniram para definir e desenvolvê-lo. Para desenvolver o protótipo, os integrantes do grupo consultaram os documentos de [Brainstorm](../desenhoSoftwareBase/brainstorm.md), [Rich Picture](../desenhoSoftwareBase/richPicture.md) e [Identidade Visual](./identidadeVisual.md) . Baseado nesses documentos foi desenvolvido no software de prototipação Figma o protótipo de alta fidelidade. Após o desenvolvimento completo de todas as telas, foi enviado a todos os membros da equipe as imagens do protótipo com a finalidade de validar todo o trabalho. Com todos os membros da equipe satisfeitos com o resultado, foi concluída a tarefa de prototipação do site.
+ 
+## 3.Resultados
```

Comment:
Ficaria interessante colocar o protótipo aqui usando o link embed pra poder haver interação

Reply:
está falando do link que o Figma disponibiliza para visualização do protótipo? Se for o caso, realmente eu esqueci de coloca-lo.

Reply:
exatamente!